api/internal/loader/fileloader_test.go

```diff
@@ -659,6 +660,35 @@ func TestLoaderHTTP(t *testing.T) {
 		_, err := l2.Load(x.path)
 		require.Error(err)
 	}
+
+	var testCaseRedirect = []testData{
+		{
+			path:            "https://example.com/resource.yaml",
+			expectedContent: "https content",
+		},
+	}
+	for _, x := range testCaseRedirect {
+		expectedLocation := "https://redirect.com/resource.yaml"
```

Comment:
It looks like `redirect.com` is owned by a company.
I understand this fakeHttpClient won't access the global internet, but I prefer to use a reserved example domain like `example.com` instead of something that someone else owns.

```# whois.verisign-grs.com

   Domain Name: REDIRECT.COM
   Registry Domain ID: 32687398_DOMAIN_COM-VRSN
   Registrar WHOIS Server: Whois.bigrock.com
   Registrar URL: http://www.bigrock.com
   Updated Date: 2023-06-20T20:24:31Z
   Creation Date: 2000-08-10T10:33:16Z
   Registry Expiry Date: 2024-08-10T10:33:15Z
   Registrar: BigRock Solutions Ltd
   Registrar IANA ID: 1495
   Registrar Abuse Contact Email: abuse@bigrock.com
   Registrar Abuse Contact Phone: +1.832-295-1535
   Domain Status: clientTransferProhibited https://icann.org/epp#clientTransferProhibited
   Name Server: NS-1118.AWSDNS-11.ORG
   Name Server: NS-1717.AWSDNS-22.CO.UK
   Name Server: NS-320.AWSDNS-40.COM
   Name Server: NS-918.AWSDNS-50.NET
   DNSSEC: unsigned
   URL of the ICANN Whois Inaccuracy Complaint Form: https://www.icann.org/wicf/
>>> Last update of whois database: 2024-06-30T18:22:52Z <<<
```

Reply:
Ок. I'll fix it!